pkg/network/proxyimpl/hybrid/proxy.go

```diff
@@ -201,8 +201,17 @@ func (p *HybridProxier) shouldEndpointsUseUserspace(endpoints *corev1.Endpoints)
 		}
 	}
 
+	// Get the Endpoints corresponding Service and checks if it has the IdledAt annotation
+	// oc idle annotates the Service object, it used to annotate the Endpoints object
+	svc, err := p.serviceLister.Services(endpoints.Namespace).Get(endpoints.Name)
```

Comment:
Does this need a role binding update? I needed `Get` rights on `services` for the o-c-m change:
- https://github.com/openshift/openshift-apiserver/pull/180

Reply:
it access the cache from the servicelister